host_classifier.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
```

Comment:
your script does not output the data in the format described here: https://encoretech.atlassian.net/wiki/spaces/EN/pages/659488782/Training+-+Bash

Reply:
updated with the correct formatting